lib/Doctrine/ORM/Query/Expr/Base.php

```diff
@@ -56,7 +56,7 @@
     protected $parts = [];
 
     /**
-     * @param array $args
+     * @param mixed $args
```

Comment:
Probably `mixed[]`?

Reply:
the Base class indeed does not enforce passing an array to `addMultiple` (and so to the constructor), as it will cast it (which will wrap scalars into an array containing them)